presto-spark-classloader-interface/src/main/java/com/facebook/presto/spark/classloader_interface/PrestoSparkShuffleSerializer.java

```diff
@@ -78,6 +83,14 @@ public DeserializationStream deserializeStream(InputStream inputStream)
             throw new UnsupportedOperationException("this method is never used by shuffle");
         }
 
+        public <T> T deserialize(byte[] array, int offset, int length, ClassTag<T> classTag)
```

Comment:
nit: should this have `@Override`?

Reply:
Unfortunately this method is not defined in the interface (in the opensource version)